README.md

```diff
@@ -27,6 +28,25 @@ This API is being used by 3 frontend projects currently being developed:
 - [Contact](#contact)
 - [License](#license)
 
+## Features
+
+- Once the App is installed user can view an onboarding screen which introduces the app and give user an idea of how it works.
+- A member gets an option to be a Mentor, Mentee or Both.
+- A member can build/customise the app profile with username, bio, skills, interests, location, occupation, etc. anytime.
+- A member can read and know more about Mentorship System including the terms and conditions, privacy and policy and code of conduct.
+- A member can directly access the AnitaB-org Github repo, Zulip chat and website from this App.
+- A member can refresh the every page.
```

Comment:
these bullet points could be removed since they are unrelated to the backend code

Reply:
Sure will do it!